www/oauth/login.php

```diff
@@ -3,19 +3,18 @@
 // Use of this source code is governed by the Polyform Shield 1.0.0 license that can be
 // found in the LICENSE.md file.
 chdir('..');
+require_once __DIR__ . '/../../vendor/autoload.php';
```

Comment:
This isn't necessary, since `common.inc` does this to start.

Reply:
Whoops.  Good catch, thanks. Done.